app/src/main/java/com/framgia/wsm/data/model/OffRequest.java

```diff
@@ -536,4 +691,147 @@ public void setOffToPeriod(String offToPeriod) {
+
+        public RequestDayOffType() {
+        }
+
+        protected RequestDayOffType(Parcel in) {
+            mId = in.readInt();
+            mSpecialDayOffSettingId = in.readInt();
+            mNumberDayOff = in.readDouble();
+        }
+
+        @Override
+        public void writeToParcel(Parcel dest, int flags) {
+            dest.writeInt(mId);
+            dest.writeInt(mSpecialDayOffSettingId);
+            dest.writeDouble(mNumberDayOff);
+        }
+
+        @Override
+        public int describeContents() {
+            return 0;
+        }
+
+        public static final Creator<RequestDayOffType> CREATOR = new Creator<RequestDayOffType>() {
+            @Override
+            public RequestDayOffType createFromParcel(Parcel in) {
+                return new RequestDayOffType(in);
+            }
+
+            @Override
+            public RequestDayOffType[] newArray(int size) {
+                return new RequestDayOffType[size];
+            }
+        };
+
+        public int getId() {
+            return mId;
+        }
+
+        public void setId(int id) {
+            mId = id;
+        }
+
+        public int getSpecialDayOffSettingId() {
+            return mSpecialDayOffSettingId;
+        }
+
+        public void setSpecialDayOffSettingId(int specialDayOffSettingId) {
+            mSpecialDayOffSettingId = specialDayOffSettingId;
+        }
+
+        public double getNumberDayOff() {
+            return mNumberDayOff;
+        }
+
+        public void setNumberDayOff(int numberDayOff) {
+            mNumberDayOff = numberDayOff;
+        }
+    }
+
+    // RequestDayOffTypesAttribute
+    public static class RequestDayOffTypesAttribute implements Parcelable {
+
+        @Expose
+        @SerializedName("special_dayoff_setting_id")
+        private String mSpecialDayOffSettingId;
+        @Expose
+        @SerializedName("number_dayoff")
+        private String mNumberDayOff;
+        @Expose
+        @SerializedName("_destroy")
+        private String mDestroy;
+
+        public RequestDayOffTypesAttribute() {
+        }
+
+        public RequestDayOffTypesAttribute(Parcel in) {
+            mSpecialDayOffSettingId = in.readString();
+            mNumberDayOff = in.readString();
+            mDestroy = in.readString();
+        }
+
+        @Override
+        public void writeToParcel(Parcel dest, int flags) {
+            dest.writeString(mSpecialDayOffSettingId);
+            dest.writeString(mNumberDayOff);
+            dest.writeString(mDestroy);
+        }
+
+        @Override
+        public int describeContents() {
+            return 0;
+        }
+
+        public static final Creator<RequestDayOffTypesAttribute> CREATOR =
+                new Creator<RequestDayOffTypesAttribute>() {
+                    @Override
+                    public RequestDayOffTypesAttribute createFromParcel(Parcel in) {
+                        return new RequestDayOffTypesAttribute(in);
+                    }
```

Comment:
ERROR: 'method def rcurly' have incorrect indentation level 20, expected level should be one of the following: 12, 16.